src/components/table-hoc/examples/TableHOCServerExamples.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import * as _ from 'underscore';
+import {ReactVaporStore} from '../../../../docs/ReactVaporStore';
+import {IDispatch, ReduxConnect} from '../../../utils/ReduxUtils';
+import {IReactVaporTestState} from '../../../utils/TestUtils';
+import {TableHeaderWithSort} from '../TableHeaderWithSort';
+import {TableHOC} from '../TableHOC';
+import {TableHOCUtils} from '../TableHOCUtils';
+import {TableRowConnected} from '../TableRowConnected';
+import {tableWithBlankslate} from '../TableWithBlankslate';
+import {tableWithFilter} from '../TableWithFilter';
+import {tableWithPagination} from '../TableWithPagination';
+import {tableWithSort} from '../TableWithSort';
+import {IExampleRowData, TableHOCServerActions} from './TableHOCServerExampleReducer';
+
+const ServerTable = _.compose(
+    tableWithBlankslate({title: 'No data caused the table to be empty'}),
+    tableWithFilter({isServer: true}),
+    tableWithBlankslate({title: 'Filter caused the table to be empty'}),
+    tableWithSort({isServer: true}),
+    tableWithPagination({isServer: true, perPageNumbers: [3, 5, 10]}),

```

Comment:
Looks a bit weird at first that `isServer` has to be passed in all of the composite components.

Is there really a use case to have `isServer: true/false` in each of those composite state?

Say, is it possible to filter server-side but then apply a sort client-side?

Reply:
@francoislg yes we have use cases of that in the Coveo's Administration Console (i.e. the Sources Table) We're not using a React component for this table yet, but someday we will.